doc/source/visualizing/geographic_projections_and_transforms.rst

```diff
@@ -0,0 +1,168 @@
+.. _geographic_projections_and_transforms:
+
+Geographic Projections and Transforms
+=====================================
+
+Geographic data that is on a sphere can be visualized by projecting that data
+onto a representation of that sphere flattened into 2d space. There exist a
+number of projection types, which can be found in the `the cartopy
+documentation <https://scitools.org.uk/cartopy/docs/latest/crs/projections.html>`_.
+With support from `cartopy <https://scitools.org.uk/cartopy/docs/latest/>`_, 
+``yt`` now supports these projection 
+types for geographically loaded data.
+Underlying data is assumed to have a transform of `PlateCarree
+<https://scitools.org.uk/cartopy/docs/latest/crs/projections.html#platecarree>`_,
+which is data on a flattened, rectangular, latitude/longitude grid. This is a
+a typical format for geographic data. 
+
+The distinction between the data transform and projection is worth noting. The data
+transform is what system your data is defined with and the data projection is
+what the resulting plot will display. For more information on this difference,
+refer to `the cartopy documentation on these differences
+<https://scitools.org.uk/cartopy/docs/latest/tutorials/understanding_transform.html>`_. 
+If your data is not of this form, feel free to open an issue or file a pull
+request on the ``yt`` github page for this feature.
+
+It should be noted that
+these projections are not the same as yt's ProjectionPlot. For more information
+on yt's projection plots, see :ref:`projection-types`.
+
+.. _install-cartopy:
+
+Installing Cartopy
+^^^^^^^^^^^^^^^^^^
+
+In order to access the geographic projection functionality, you will need to have an
+installation of ``cartopy`` available on your machine. If you're using conda as
+your package manager, this should be sufficient:
+
+.. code-block:: bash
+
+    conda install cartopy
+
+If you're on a mac and are using pip, there can be conflicts with the GEOS
+library and cartopy / cartopy dependencies. To avoid these issues use your
+package manager of choice to install proj4 and geos (``proj`` and ``geos``).
+Following that, build cartopy and shapely from source.
+For example, a user using homebrew and pip
+would execute the following commands:
+
+.. code-block:: bash
+
+    brew install proj geos
+    brew upgrade proj geos
+    pip install --no-binary :all: shapely cartopy
+
+
+On ubuntu you'll need to install the following packages: ``libproj-dev``,
+``proj-data``, ``proj-bin``, and ``libgeos-dev``.
+
+Using Basic Transforms
+^^^^^^^^^^^^^^^^^^^^^^^
+
+As mentioned above, the default data transform is assumed to be of `PlateCarree
+<https://scitools.org.uk/cartopy/docs/latest/crs/projections.html#platecarree>`_,
```

Comment:
This is a very minor note. There are multiple places in this doc file with a link where the link text is "PlateCarree." This will cause sphinx to think a label has multiple definition and issue a warning. This can be fixed by making the trailing underscore into two underscores, e.g.:
```
`PlateCarree
<https://scitools.org.uk/cartopy/docs/latest/crs/projections.html#platecarree>`__
```

Reply:
Ah, thanks for that @brittonsmith! I really appreciate the explanation too. It should be fixed now.